examples/cim/dqn/dist_actor.py

```diff
@@ -21,25 +21,33 @@
     if config.experience_shaping.type == "truncated":
         experience_shaper = TruncatedExperienceShaper(**config.experience_shaping.truncated)
     else:
-        experience_shaper = KStepExperienceShaper(reward_func=lambda mt: 1-mt["container_shortage"]/mt["order_requirements"],
-                                                  **config.experience_shaping.k_step)
+        experience_shaper = KStepExperienceShaper(
+            reward_func=lambda mt: 1-mt["container_shortage"]/mt["order_requirements"],
+            **config.experience_shaping.k_step
+        )
 
-    exploration_config = {"epsilon_range_dict": {"_all_": config.exploration.epsilon_range},
-                          "split_point_dict": {"_all_": config.exploration.split_point},
-                          "with_cache": config.exploration.with_cache
-                          }
+    exploration_config = {
+        "epsilon_range_dict": {"_all_": config.exploration.epsilon_range},
```

Comment:
_all_ for what?

Reply:
Explorer class has been revamped. This issue no longer exists.